### DISCUSSION TITLE
Content lost after updating component name?

### DISCUSSION THREAD

**Discussion by mhedengren** [strapi/strapi#6491] [Legacy Topics]
Hi!
I have a quick question.

I have a content type that uses a repeatable component called **"description"**, but i figured I wanted to **change it to simply "desc"**. However upon doing so and saving, all the previously added **data for that component in my collection is now gone**.

I mean imagine you made a typo and wanted to change that, is there no way of doing that without losing all of your collection-data for that component?

Many thanks.

**Comment by sunnysonx** [comment]
> I have a content type that uses a repeatable component called **"description"**, but i figured I wanted to **change it to simply "desc"**. However upon doing so and saving, all the previously added **data for that component in my collection is now gone**.

Data is not deleted from DB. Check in DB if you have your old component `components_description` with all the data in it.
If you have it, then edit your new component and set a custom collectionName to it.

**Reply by mhedengren** [reply]
Sorry maybe I was a bit unclear in my first post. I actually didn't edit the component name itself but rather the name of the component when imported to the Content-Types builder.

I'm not really sure where I find this data in the strapi-api folder though. Sorry I'm a bit fresh to this.

**Reply by mhedengren** [reply]
I see that strapi creates new tables/collection when you rename a field. How do I migrate the data that was in the old field to the new? Any idea?
